apps/frontend/src/components/chat/chat-message/chat-citations/web-citation-item.tsx

```
@@ -1,17 +1,35 @@
-function faviconUrlForHostname(hostname: string) {
-	return `https://www.google.com/s2/favicons?domain=${encodeURIComponent(hostname)}&sz=32`;
-}
-
 export function WebCitationItem({ source }: { source: WebCitationSource }) {
 	const hostname = new URL(source.url).hostname;
-	const [iconSrc, setIconSrc] = useState(() => faviconUrlForHostname(hostname));
+	const [iconSrc, setIconSrc] = useState(GLOBE_ICON_SRC);
+
+	useEffect(() => {
+		const token = useAuthStore.getState().session?.access_token;
+		if (!token) return;
```

ESLint

[error] Expected { after 'if' condition.

(curly)